server/config.ts

```diff
@@ -15,30 +15,36 @@ const nonSensitiveSharedConfig = {
 }
 
 const keysToPullFromEnv = [
-  'SophtronClientId',
-  'SophtronClientSecret',
-
-  'UcpAuthClientId',
-  'UcpAuthClientSecret',
-  'UcpAuthEncryptionKey',
-
-  'CryptoKey',
-  'CryptoIv',
-
-  'Demo',
-  'DefaultProvider',
-  'Port',
-
-  'HostUrl',
-  'WebhookHostUrl',
-  'LogLevel',
-
-  'Env',
-
-  'RedisServer',
-  'RedisCacheTimeSeconds',
-  'ResourcePrefix',
-  'ResourceVersion'
+  "CryptoKey",
+  "CryptoIv",
+
+  "AuthProvider", //supported: ucp, local, sophtron
```

Comment:
We've been putting comments describing these variables in the .env.example file instead of here